source/meetings/2016/november/index.html.md

```diff
@@ -0,0 +1,97 @@
+---
+updated_by:
+  email: murray.steele@gmail.com
+  name: Murray Steele
+created_by:
+  email: murray.steele@gmail.com
+  name: Murray Steele
+category: meeting
+title: November 2016 Meeting
+updated_at:
+published_at: 2016-10-17 13:20:16 +0100
+created_at: 2016-10-17 12:54:32 +0100
+parts: {}
+status: Published
+---
+
+The November 2016 meeting of LRUG will be on *__Tuesday__ the 15th of November*, from
```

Comment:
You're probably right; based on confusion folk had about October being 45 minutes later than usual the day being bold won't be enough.  I'll call it out more, and make sure to do the same in the email and tweet announcements.